.github/workflows/coverity.yml

```diff
@@ -156,44 +156,55 @@ jobs:
         run: |
           rm -rf ${COVERITY_TOOL_DIR} && mkdir -p ${COVERITY_TOOL_DIR}
           pushd ${COVERITY_TOOL_DIR}
-            wget https://scan.coverity.com/download/linux64 --progress=bar:force:noscroll --post-data "token=${{ secrets.COVERITY_TOKEN }}&project=openvino" -O coverity_tool.tgz
-            tar -I pigz -xf coverity_tool.tgz && rm coverity_tool.tgz
+            wget https://ubit-artifactory-or.intel.com/artifactory/coverity-or-local/Enterprise/license.dat -O license.dat
+            wget https://ubit-artifactory-or.intel.com/artifactory/coverity-or-local/Enterprise/cov-analysis-linux64-2024.6.1.sh -O cov-analysis-2024.6.1.sh
+            chmod u+x cov-analysis-2024.6.1.sh
+            ./cov-analysis-2024.6.1.sh --unattended --skip-license --prefix ${COVERITY_TOOL_DIR}
+            ./cov-analysis-2024.6.1.sh \
+              -q \
+              --installation.dir=${COVERITY_TOOL_DIR} \
+              --license.agreement=agree \
+              --license.region=0 \
+              --license.type.choice=0 \
+              --license.cov.path=${COVERITY_TOOL_DIR}/license.dat \
+              --component.sdk=false \
+              --component.skip.documentation=true
+            rm cov-analysis-2024.6.1.sh
+            export PATH=$PATH:${COVERITY_TOOL_DIR}/bin/
           popd
 
-      - name: Cmake build - OpenVINO with Coverity
-        run: ${COVERITY_TOOL_DIR}/cov-analysis*/bin/cov-build --dir ${BUILD_DIR}/cov-int cmake --build ${BUILD_DIR} --parallel $(nproc) --config ${{ env.CMAKE_BUILD_TYPE }}
+      - name: Cmake build - OpenVINO with Coverity (create coverity artifacts)
+        run: cov-build --dir ${BUILD_DIR}/cov-int cmake --build ${BUILD_DIR} --parallel $(nproc) --config ${{ env.CMAKE_BUILD_TYPE }}
 
-      - name: Pack Artefacts
-        run: tar -cvf - cov-int | pigz > openvino.tgz
-        working-directory: ${{ env.BUILD_DIR }}
+      - name: Coverity - analyze created artifacts
+        run: |
+          cov-manage-emit --dir cov-int --tu-pattern "file('/usr/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/thirdparty/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/src/plugins/intel_gpu/thirdparty/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/src/plugins/intel_cpu/thirdparty/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/src/plugins/intel_npu/thirdparty/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/build/.*')" delete
+          cov-manage-emit --dir cov-int --tu-pattern "file('/docs/.*')" delete
+          cov-analyze \
+          --dir ${BUILD_DIR}/cov-int \
+          --concurrency \
+          --security \
+          --rule --enable-constraint-fpp --enable-fnptr --enable-virtual --enable-default --webapp-security --enable-audit-checkers --enable-audit-mode \
+          --strip-path ${{ env.OPENVINO_REPO }}
```

Comment:
strip-path is very important for us - this setting will decide if exclusion made in lines: 181-187 will work.